manager/catalog_manager.go

```diff
@@ -3,6 +3,9 @@ package manager
 import (
 	"flag"
 	"fmt"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Typically, you want to see these sorted like they were... the gopkg and github separated by a line from the builtin Golang libs.